### PR TITLE
update to 0.86.0

### PR DESCRIPTION
**Pull request by fdncred**
This PR updates the plugin to nushell 0.86.0

### CODE REVIEW

File: src/theme.rs [hunk 1]
```diff
@@ -21,17 +21,16 @@ impl From<ThemeDescription> for Value {
...
+                    None => Value::test_nothing(),
                 },
                 "author" => match author {
-                    Some(author) => Value::string(author, Span::unknown()),
-                    None => Value::nothing(Span::unknown()),
+                    Some(author) => Value::test_string(author),
+                    None => Value::test_nothing(),
                 },
-                "default" => Value::bool(default, Span::unknown()),
+                "default" => Value::test_bool(default),
```
Comment (cptpiepmatz) [inline src/theme.rs:33]: According to the docs, these `Value::test_*` should only be used for tests. Is the naming then just a bit odd if they are just values with unknown span?

@fdncred

Comment (fdncred) [reply src/theme.rs:33]: `Value::test_blah()` is just a shorthand way of providing a `Span::unknown()` automatically. It's identical under the hood. So, if you have no span, that's what you should do.

Comment (cptpiepmatz) [reply src/theme.rs:33]: Then it's a weird name in my opinion.

Comment (fdncred) [reply src/theme.rs:33]: It was originally intended for tests, but it still works for this purpose.

**Pull request merged**